mdtraj/core/trajectory.py

```diff
@@ -1669,6 +1669,76 @@ def remove_solvent(self, exclude=None, inplace=False):
 
         return self.atom_slice(atom_indices, inplace = inplace)
 
+    def smooth(self, width=11, order=3, atom_indices=None, inplace=False):
+        """
+        Smoothen a trajectory using a zero-delay Buttersworth filter.
+        This code is copied from the scipy cookbook, with sytlistic improvements.
+        http://www.scipy.org/Cookbook/FiltFilt
+        
+        Parameters
+        ----------
+        width : float
```

Comment:
Whatever you think is best re: int or float, just make sure the docstring is correct

I don't think it should have a default value though